Fix export dialog sizing issue on small devices

m4gr3d opened the pull request:
Fixes https://github.com/godotengine/godot/issues/73130

- Update the min height from `700` to `500` to match the editor's min height of `600`
- Wrap the `Resources` and `Encryption` sections of the `TabContainer` within a `ScrollContainer`. This prevents the export dialog from trying to resize to fit the tallest of those sections, and make them accessible on small screen devices.

<!--
Please target the `master` branch in priority.

Relevant fixes are cherry-picked for stable branches as needed by maintainers.

To speed up the contribution process and avoid CI errors, please set up pre-commit hooks locally:
https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html
-->

m4gr3d committed:
Fix export dialog sizing issue on small devices

Fixes https://github.com/godotengine/godot/issues/73130

- Update the min height from `700` to `500` to match the editor's min height of `600`
- Wrap the sections of the `TabContainer` within a `ScrollContainer`. This prevents the export dialog from trying to resize to fit the tallest of those sections, and make them accessible on small screen devices.

KoBeWi commented:
Another thing that could be scrollable is error list. It can get quite lengthy 🤔
![image](https://github.com/godotengine/godot/assets/2223172/708c1c79-02f6-44d0-9fbe-09630f1cdbdf)

Hicham12312 commented:
How do I compile this to an apk so i can use it?

KoBeWi commented:
The errors are not visible at all now 🙃
ScrollContainer has no size by default, you should probably give it a minimum one.
The file Tree still needs minimum size.

m4gr3d commented:
> Since this tab is scrollable now, the Tree can get a minimum size. It's ridiculously small.

@KoBeWi I've added a min size for the file tree, so it's now visible when the dialog is resized.

> Another thing that could be scrollable is error list. It can get quite lengthy

I've reverted the changes to the error list; I couldn't figure out the proper configuration that would get it to both be scrollable, but expand as needed based on its parent's height.
Also making the error list scrollable is not needed to address https://github.com/godotengine/godot/issues/73130; the current updates already make the project export dialog usable on small screens even with multiple errors showing.

akien-mga commented:
Thanks!
